megamek/src/megamek/common/AmmoType.java

```diff
@@ -13457,12 +13464,14 @@ public AmmoType createMunitionType(AmmoType base) {
                     || (munition.getAmmoType() == AmmoType.T_MML) || (munition.getAmmoType() == AmmoType.T_NLRM))
                     && (munition.getMunitionType() == AmmoType.M_SEMIGUIDED)) {
                 cost *= 3;
+                bv *= 1;
             }
 
             if (((munition.getAmmoType() == AmmoType.T_LRM) || (munition.getAmmoType() == AmmoType.T_LRM_IMP)
                     || (munition.getAmmoType() == AmmoType.T_MML) || (munition.getAmmoType() == AmmoType.T_NLRM))
                     && (munition.getMunitionType() == AmmoType.M_SWARM)) {
                 cost *= 2;
+                bv *= 1;
```

Comment:
multiplying by 1 doesn't do anything?

Reply:
Reminder that the BV has been checked. Beats have a blank line with no idea that it's been compared to source.

Reply:
For be its easier to see the missing BV formula than read the comment for everyone. If I know they are missing a BV multiplier then I know we have an issue of some type.